Tasks/Kasinskiy/Task1/task1.py

```diff
@@ -0,0 +1,9 @@
+from decimal import Decimal as dD
+
+initial_deposit = 20000
+interest_rate = 15
+deposit_term = 60
+
+final_income = dD(dD(initial_deposit)*(1+dD(interest_rate)/(100*12))**dD(deposit_term))
```

Comment:
too much type converting, to store the main variables as decimal values would be enough